neos/neos-development-collection
# BUGFIX: Synchronize react-ui menu state with vanilla js

The new vanilla js menu did use the session storage to save the states and it turns out that the react-ui uses the local storage. This patch synchronizes the states and takes care that when you change a menu state in the content module (react-ui) the state is also available in the user module for instance.

**What I did**
Use also the local storage in the vanilla js based menu and adopt the data structure to behave like the react-ui.

**How to verify it**
Change menu states in the content module an check then if the state is also in the user module. And vice versa.

Fixes: #2682
